tools/icu/README.md

```diff
@@ -27,7 +27,7 @@ internationalization functionality.
 
 ```shell
 ./configure \
-    --with-intl=small-icu \
+    --with-intl=full-icu \
     --with-icu-source=http://download.icu-project.org/files/icu4c/58.1/icu4c-58_1-src.tgz
```

Comment:
Update url to: http://download.icu-project.org/files/icu4c/64.2/icu4c-64_2-src.tgz.